packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -2324,6 +2313,16 @@ async function handler(
         );
       }
     }
+  } else {
+    // If isConfirmedByDefault is false, then booking can't be considered ACCEPTED and thus EventManager has no role to play. Booking is created as PENDING
+    loggerWithEventDetails.debug(
+      `EventManager doesn't need to create or reschedule event for booking ${organizerUser.username}`,
+      safeStringify({
+        calEvent: getPiiFreeCalendarEvent(evt),
+        isConfirmedByDefault,
+        paymentValue: paymentAppData.price,
+      })
+    );
```

Comment:
looks good, we might want considering rewriting this if / else if / else block in a separate function with early returns to make it more readable